src/chopper_count.cpp

```diff
@@ -12,56 +12,86 @@ namespace chopper::count
     parser.add_option(config.k,
-                      'k', "kmer-size",
-                      "The kmer size to count minimisers.");
+                      '\0', "kmer-size",
+                      "The k-mer size influences the estimated counts. Choosing a k-mer size that is too small for your "
+                      "data will result in files appearing more similar than they really are. Likewise, a large k-mer "
+                      "size might miss out on certain similarities. For DNA sequences, a k-mer size between [?,?] has" // range??
+                      "proven to work well.");
 
     parser.add_option(config.w,
-                      'w', "window-size",
-                      "The window size for minimisers.");
+                      '\0', "window-size",
+                      "Instead of looking at all k-mers, one can only store a representative set of k-mers, e.g. "
+                      "minimizers. Their number is proportional to that of all k-mers in a sequence but the total "
+                      "amount is less, resulting in lower memory consumption (RAM).");
 
     parser.add_option(config.sketch_bits,
-                      's', "sketch-bits",
+                      '\0', "sketch-bits",
                       "The number of bits the HyperLogLog sketch should use to distribute the values into bins.",
                       seqan3::option_spec::standard,
                       seqan3::arithmetic_range_validator{5, 32});
 
     parser.add_flag(config.disable_sketch_output, '\0', "disable-sketch-output",
-                    "Although the sketches will improve the layout you might want to disable this for saving disk "
-                    "space by not writing out sketch files. Note that you cannot use the options --estimate-unions nor "
-                    "--rearrange-bins in chopper layout then.");
+                    "Although the sketches will improve the layout, you might want to disable "
+                    "writing the sketch files to disk. Doing so will save disk space. However, you cannot use neither "
+                    "--estimate-unions nor --rearrange-user-bins in chopper layout without the skectes. Note that "
+                    "this option does not decrease run time as sketches have to be computed either way.");
 
     parser.add_flag(config.disable_minimizers, '\0', "disable-minimizers",
-                    "Compute pure kmer counts instead of minimizers. Note that selecting -k == -w would not be enough "
-                    "because the minimizer hash will still consider the reverse complement and thus differ from a "
-                    "pure kmer count.");
+                    "Compute pure k-mer counts instead of minimizers. Note that selecting k-mer-size == window-size is "
+                    "not enough because the minimizer hash will still consider the reverse complement and thus differ "
+                    "from a pure k-mer count (called canonical k-mers).");
```

Comment:
Didn't I have a suggestion for that? 🤔 
Even though we might remove the option altogether.

Reply:
I removed the minimzier option now